client/state/action-types.js

```diff
@@ -125,6 +125,9 @@ export const JETPACK_SYNC_START_ERROR = 'JETPACK_SYNC_START_ERROR';
 export const JETPACK_SYNC_STATUS_REQUEST = 'JETPACK_SYNC_STATUS_REQUEST';
 export const JETPACK_SYNC_STATUS_SUCCESS = 'JETPACK_SYNC_STATUS_SUCCESS';
 export const JETPACK_SYNC_STATUS_ERROR = 'JETPACK_SYNC_STATUS_ERROR';
+export const LAYOUT_SET_FOCUS = 'LAYOUT_SET_FOCUS';
```

Comment:
I think these should be `LAYOUT_FOCUS_SET` and `LAYOUT_NEXT_FOCUS_SET` (the third seems correct)